readme changes for mac, and gitignore for tool installation

jacobhogrefe opened the pull request:
i added a section for mac developers in the README file. i also changed the gitignore to not ignore the .config folder anymore. this makes the dotnet restore command work easy to install all of the tools/libraries easier

a-r-t commented on .gitignore:
```diff
@@ -72,5 +72,5 @@ obj/
 
 todo.txt
-.config/
+# .config/
```
We need to ignore the `.config/`, these are machine specific settings

a-r-t commented on Monogame-RPG-Engine/.config/dotnet-tools.json:
```diff
@@ -0,0 +1,36 @@
+{
```
Please remove this file

a-r-t commented on README.md:
```diff
@@ -19,6 +19,27 @@ You may need to install/update .NET Framework for this.
 
 After opening the project successfully, you should be able to run the game and play my fun little cat RPG identical to the Java version.
 
+### For Mac Developers
```
Can you make a subsection for windows developers like you did here for Mac, just to make it more clear to someone trying to run it that there are OS specific instructions? Wouldn't want someone with a Mac to find the setup instructions and follow line by line only to realize it won't work and then later hit the Mac instructions...I've seen it happen so many times while teaching lol.

a-r-t commented:
Thank you so much for working to get this running on your Mac and adding documentation on how to do it! I just have a few comments for you before I merge this in!

jacobhogrefe commented:
Of course! I changed the README to reflect the comments you made. There is now a link to the Windows/Mac instructions, and links to install both Visual Studio and .NET. Most of your instructions were kept for the Windows section. As for the Mac instructions, I updated them to essentially make the `dotnet-tools.json` through commands only.

I will update again with proper build instructions for Windows sometime soon.
